packages/admin/src/common/ClearInputButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { ButtonBase, ButtonBaseProps, Theme, WithStyles } from "@material-ui/core";
+import { Clear as ClearIcon } from "@material-ui/icons";
+import { createStyles, withStyles } from "@material-ui/styles";
+import * as React from "react";
+
+import { getDefaultVPAdminInputStyles } from "../form";
+
+export interface ClearInputButtonThemeProps {
+    customIcon?: (disabled?: boolean) => React.ReactNode;
```

Comment:
shouldn't that be just `icon`? every prop is somehow "custom"

Reply:
That's true 😄